.github/workflows/prcheck.yml

```
@@ -0,0 +1,104 @@
+name: PR Check
+
+on:
```

YAMLlint (1.38.0)

[warning] truthy value should be one of [false, true]

(truthy)